packages/snaps-controllers/src/interface/SnapInterfaceController.test.tsx

```diff
@@ -1164,6 +1434,94 @@ describe('SnapInterfaceController', () => {
         ),
       ).rejects.toThrow('Interface not created by foo.');
     });
+
+    it('can select an account owned by the snap', async () => {
```

Comment:
Duplicate test?

Reply:
Nope, since `snapOwnsAccount` is defined in `createInterface` and `updateInterface` we need to test that case 2 times in order to get coverage

Reply:
We can't just rely on one of the two controller methods to get full coverage for `snapOwnsAccount`

Reply:
Ah, sorry I had missed that this was for another method